.env-n8n.example

```diff
@@ -1,6 +1,6 @@
 # n8n
-DB_POSTGRESDB_USER=n8n
-DB_POSTGRESDB_PASSWORD=password
+# DB_POSTGRESDB_USER=n8n
+# DB_POSTGRESDB_PASSWORD=password
```

Comment:
Why comment these out?

Reply:
These are added as a part of the Postgres/README.md instructions